network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CollectionCache.java

```diff
@@ -79,6 +79,31 @@ public class CollectionCache<T extends IdentifiableAttributes> {
      */
     private final Map<String, Set<String>> removedExtensionAttributes = new HashMap<>();
 
+    /**
```

Comment:
loadflow fails after a line deletion : 
gridstudy-loadflow-server-1  | Caused by: com.powsybl.ws.commons.computation.ComputationException: loadflow has failed: Cannot manipulate operational limits groups for branch (MTMARL31VALAU) as it has not been loaded into the cache.